courses/views.py

```diff
@@ -38,6 +41,44 @@ class ProgramViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = ProgramSerializer
 
 
+class ProgramLearnersView(APIView):
```

Comment:
I am modifying the returned list slightly, making it a dictionary. Not sure how I would use `ListAPIView`